scripts/install

```diff
@@ -80,66 +102,19 @@ sudo chmod 644 /lib/systemd/system/cjdns-resume.service
 sudo systemctl daemon-reload
 sudo systemctl enable cjdns.service
 
-# WiFi Access Point on RPi3
-if [[ $RPI_REVISION == *"a02082"* || $RPI_REVISION = *"a22082"* ]] && [ ! -z "$WITH_WIFI_AP" -a "$WITH_WIFI_AP" = "true" ]; then
-    # Install hostapd and dnsmasq to run IEEE 802.11 Access Point
-    if ! [ "$(which hostapd)" ] || ! [ "$(dnsmasq)" ]; then
-        sudo apt-get update
-        sudo apt-get install hostapd dnsmasq -y
-    fi
+# 802.11s Mesh Point interface
+if [ ! -z "$WITH_MESH_POINT" -a "$WITH_MESH_POINT" == "true" ]; then
+    source prototype-cjdns-pi2/scripts/mesh-point/install
```

Comment:
You'd be running this off master (without the file) unless you run from the commit before this change: https://github.com/tomeshnet/prototype-cjdns-pi2/pull/44/commits/cb7df6cdcec0def11f7d3b0185185340f334d634